src/api/address.ts

```diff
@@ -10,14 +12,17 @@ export type Address = {
 };
 
 export const fetchAddress = async (hash: string) => {
-  const response = await fetch(
-    `https://www.stargazer.network/api/v1/address/${hash}`
-  );
-  const payload = await response.json();
+  const response = await fetch(`${AppEnv.STAR_GAZER_API}/address/${hash}`);
```

Comment:
I'd suggest to pass in environment variables via webpack's DefinePlugin. This already happens with `PUBLIC_PATH`, so you could just add yours in `webpack.config.js`. If needed these can also be parameterized by the environment which is perfectly known at compilation time. `process.env` is because it's commonly used for this purpose and also because Typescript declares it per default. Furthermore, this way we keep our source clean from environment stuff.